Campus Density/DiningCells/MealFilterCell.swift

```diff
@@ -69,19 +79,19 @@ class MealFilterCell: UICollectionViewCell {
         }
         filterButtons.removeAll()
 
-        for (index, meal) in mealModel.meals.enumerated() {
+        for meal in mealModel.meals {
             let button = UIButton()
-            button.tag = index
```

Comment:
The button tag is used to control which meal the filter selects when the button is pressed, and removing this line makes all the buttons set the meal to breakfast/other first meal when tapped